salt/netapi/rest_cherrypy/app.py

```diff
@@ -8,6 +8,7 @@
 .. py:currentmodule:: salt.netapi.rest_cherrypy.app
 
 :depends:   - CherryPy Python module
+            - salt-api binary
```

Comment:
I think the word "executable" might be more appropriate here than "binary", since it's really just a script that's running and not a binary file, strictly speaking. :]